src/CSModule.sol

```diff
@@ -871,8 +928,7 @@ contract CSModule is
         external
         onlyRole(STAKING_ROUTER_ROLE)
     {
-        // skip all existing queue batches to effectively wipe out the queue
-        depositQueue.head = depositQueue.tail;
+        // Nothing to do.
```

Comment:
Fixed separately. Remove to avoid merge conflicts

Reply:
Can't do so, because `depositQueue` is gone.